pkgs/development/compilers/flutter/default.nix

```diff
@@ -0,0 +1,30 @@
+{ callPackage }:
+
+let
+  mkFlutter = opts: callPackage (import ./flutter.nix opts) { };
+  getPatches = dir:
+    let files = builtins.attrNames (builtins.readDir dir);
+    in map (f: dir + ("/" + f)) files;
+in {
+  stable = mkFlutter {
+    pname = "flutter";
+    channel = "stable";
+    version = "1.12.13+hotfix.8";
+    sha256Hash = "01ik4xckr3fp65sq4g0g6wy5b9i0r49l643xmbxa6z9k21sby46d";
+    patches = getPatches ./patches/stable;
```

Comment:
Can you symlink patches that are the same?

Reply:
Pushed a fix, there is only one patch that is common.